test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Tests.ps1

```diff
@@ -0,0 +1,247 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
```

Comment:
The file name doesn't contain `Module`. Maybe `ModuleCompatiblePSEditions.Tests.ps1`?